### PR TITLE
DibiFluent && Dibi\Fluent support

### PR DESCRIPTION
**Pull request by cniry**

### CODE REVIEW

File: src/AggregationFunction/FunctionSum.php [hunk 1]
```diff
@@ -63,7 +62,7 @@ public function getFilterDataType()
 	 */
 	public function processDataSource($dataSource)
 	{
-		if ($dataSource instanceof DibiFluent) {
+		if ($dataSource instanceof Dibi\Fluent || $dataSource instanceof DibiFluent) {
```
Comment (paveljanda) [inline src/AggregationFunction/FunctionSum.php]: Will that work when there is no `use DibiFluent;` statement in the file header?

Comment (paveljanda) [reply src/AggregationFunction/FunctionSum.php]: @cniry ping

Comment (elring) [reply src/AggregationFunction/FunctionSum.php]: I think it is at least missing "use Nette" OR the test should be "$dataSource instanceof \Dibi\Fluent ..." (without any of these changes it doesn't work for me).

**Comment by f3l1x** [comment]
Duplicate? https://github.com/contributte/datagrid/pull/662

**Comment by f3l1x** [comment]
It needs to be rebased, @cniry.

**Comment by paveljanda** [comment]
@cniry ping

**Pull request closed without merging**